libs/partners/google-vertexai/tests/unit_tests/test_chat_models.py

```diff
@@ -202,3 +215,104 @@ def test_default_params_gemini() -> None:
         message = HumanMessage(content=user_prompt)
         _ = model([message])
         mock_start_chat.assert_called_once_with(history=[])
+
+
+@pytest.mark.parametrize(
```

Comment:
nits: I'm not sure we need parametrize here. it might be slightly more readable without it.

Reply:
i use it for testing different response format from as the candidate.